src/components/buttons/AddPetBtn.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Button } from '@/components/ui/button';
+import { TbCirclePlus } from 'react-icons/tb';
+
+export const AddPetBtn = () => {
+  return (
+    <Button className="gap-2">
+      <TbCirclePlus /> Add a new pet
+    </Button>
+  );
+};
+
```

Comment:
The issue with this button and the modal it invokes is how the modal component from ShadCN is invoked.
in the modal component there is 
```jsx
<DialogTrigger asChild>
  <AddPetBtn />
</DialogTrigger>
```
So the modal contains a button that triggers it.
Then on the home page, we have a `<AddPetModal />`, not a button. This can be confusing but it is how ShandCN modal component works.

I tried to modularize the code by creating a separata button component for this.